adhocracy-plus/templates/a4dashboard/base_project_list.html

```diff
@@ -21,10 +21,6 @@
 
         {% has_perm 'a4_candy_organisations.change_organisation' request.user view.organisation as user_may_change_organisation %}
         {% if user_may_change_organisation %}
-            <a href="{% url 'organisation-settings' organisation_slug=view.organisation.slug %}"
-               class="btn btn--light btn--full">
-                {% trans 'Edit your organisation' %}
-            </a>
             {% if settings.a4_candy_cms_settings.ImportantPages.manual_link %}
```

Comment:
Nothing to do with your PR, but we should show the link to everyone that can access the dashboard, not only to the ones that are allowed to change the organisation.

Reply:
Ah, let's do that later! We'll look at the dashboard for the whole sprint now!

Reply:
ok i'm doing some last cleaning. why should people that can't click on the link be able to see it though? or am i mis-understanding the permissions?

Reply:
The link only links to the manual. Our a+-wiki page. Anyone should be able to see it.
I think the permission was there for the organisation update link (where it makes sense), but wasn't closed in the right place. Or is there another link even after that?